IR_Turret/IR_Turret.ino

```diff
@@ -1,8 +1,9 @@
 #include <Arduino.h>
 #include <Servo.h>
 
-#include "IR_Signals.h"
-#include "IR_Logic.h"
+#define DECODE_NEC
```

Comment:
Not sure if this is exactly the right path, but I took this from the IRremote examples in the Arduino IDE.